openreview/tools.py

```diff
@@ -19,20 +19,22 @@
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
 
-def decision_to_venue(venue_id, decision_option):
+def decision_to_venue(venue_id, decision_option, accept_options=None):
     """
     Returns the venue for a submission based on its decision
 
     :param venue_id: venue's short name (i.e., ICLR 2022)
     :type venue_id: string
     :param decision_option: paper decision (i.e., Accept, Reject)
     :type decision_option: string
+    :param accept_options: accept decisions (i.e., [ Accept (Best Paper), Invite to Archive ])
+    :type accept_options: list
     """
     venue = venue_id
-    if 'Accept' in decision_option:
-        decision = decision_option.replace('Accept', '')
-        decision = re.sub(r'[()\W]+', '', decision)
-        if decision:
+    decision = re.sub(r'[()\W]+', '', decision_option)
```

Comment:
I think this can be inside the if, right?

Reply:
it seems the condition `if (not accept_options and 'Accept' in decision_option) or (accept_options and decision_option in accept_options)` is used in several places, can we create a tools function that we use from the venue class, process function and this function too?

Reply:
Yes that would be great, updated to add the tools function